build.py

```diff
@@ -80,6 +80,7 @@ def __env(envdir):
 def env():
     """set up a base virtual environment"""
     _env()
+    _deps()
```

Comment:
Why? I want the dependencies to be installed when I build the environment, don't I?